upstream.wrap

```diff
@@ -0,0 +1,6 @@
+[wrap-file]
+directory = lz4-1.9.3
+
+source_url = https://github.com/lz4/lz4/archive/v1.9.3.tar.gz
+source_filename = lz4-1.9.3.tgz
+source_hash = 030644df4611007ff7dc962d981f390361e6c97a34e5cbc393ddfbe019ffe2c1
```

Comment:
Hm. I added it, but I'm unable to make things work without the `liblz4 = liblz4_dep` line. :/ (At least when trying to include it with `dependency('liblz4', required: get_option('ENABLE_LZ4_EXT'), static: get_option('default_library') == 'static')`

Reply:
That's the correct way to do it, you can just remove dependency_names here and remove meson.override_dependency from the meson.build (since it is not available in the upstream meson.build, and even if it was, it only works with very recent versions of meson).

Note also, you must remove `program_names = lz4`, because this fails by default since the lz4 executable is not built or defined.

Reply:
Ah, so just a section `[provide] {pkgconfig-name} = {variable-containing-dependency-object}` is enough. kk, I'll apply the same to rdkafka and yajl.

> Note also, you must remove program_names = lz4, because this fails by default since the lz4 executable is not built or defined.
[It's](https://github.com/lz4/lz4/blob/d44371841a2f1728a3f36839fd4b7e872d0927d3/contrib/meson/meson/programs/meson.build#L30) not?